### PR TITLE
[HOPSWORKS-1356] Changing to the executions API and job configuration

### PR DESCRIPTION
**Pull request by tkakantousis**

### CODE REVIEW

File: hops/featurestore.py [hunk 1]
```diff
@@ -1398,7 +1403,12 @@ def import_featuregroup_redshift(storage_connector, query, featuregroup, primary
     arguments = locals()
     arguments['type'] = "REDSHIFT"
     core._do_import_featuregroup(json.dumps(arguments))
-    job.launch_job(featuregroup)
+    #path to json file in hdfs
+    input_json_path = '--job_spec hdfs:///Projects/' + \
+                      os.environ[constants.ENV_VARIABLES.HOPSWORKS_PROJECT_NAME_ENV_VAR] + \
+                      '/Resources/featurestore-trainingdataset-job/configurations/' + \
```
Comment (moritzmeister) [inline hops/featurestore.py]: Should also be `'/Resources/featurestore-import/configurations/'` like above for the S3 case.

**Review by moritzmeister — CHANGES_REQUESTED** [review]
There is one copy paste error, that needs to be changed.

File: hops/featurestore.py (hunk 1 above)
Comment (tkakantousis) [reply hops/featurestore.py]: Fixed thanks

File: hops/featurestore.py [hunk 2]
```diff
@@ -1314,7 +1314,12 @@ def import_featuregroup_s3(storage_connector, featuregroup, path=None, primary_k
     arguments = locals()
     arguments['type'] = "S3"
     core._do_import_featuregroup(json.dumps(arguments))
-    job.launch_job(featuregroup)
+    #path to json file in hdfs
+    input_json_path = '--job_spec hdfs:///Projects/' + \
```
Comment (tkakantousis) [inline hops/featurestore.py:1318]: @SirOibaf Is it ok to have `hdfs` scheme hard-coded? We are not using hopsfs in Hopsworks if I'm not mistaken

Comment (SirOibaf) [reply hops/featurestore.py:1318]: Yes that can work.

**Review by moritzmeister — APPROVED** [review]
:+1:

**Pull request merged**